src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -207,11 +222,23 @@ export class CalcitePickListItem {
     );
   }
 
+  renderRemoveAction() {
+    if (!this.removable) {
+      return null;
+    }
+    return (
+      <calcite-action class={CSS.remove} text="remove" onClick={this.removeClickHandler}>
```

Comment:
Can we add a prop for the `remove` text` and store the value in the resources file as a TEXT const?